src/modules/common/dto/mentorfilters.dto.ts

```diff
@@ -1,12 +1,14 @@
 import { ApiModelPropertyOptional } from '@nestjs/swagger';
 import { IsOptional } from 'class-validator';
+import { Transform } from 'class-transformer';
 import { PaginationDto } from './pagination.dto';
 
 export class MentorFiltersDto extends PaginationDto {
-  // Private fields used internally only
+  @ApiModelPropertyOptional()
+  @IsOptional()
+  @Transform((val: string) => val === 'true')
```

Comment:
Let me know if we should change this.

Reply:
The client sends the `available` param in the query string — it'll be a string. I had to explicitly add the transform to convert it to a boolean. If we add `@IsBoolean`, a request to `/mentors?available=true` will fail with a 400.

Do let me know if I'm missing something.

Reply:
Thanks for explaining!  I didn't realize that the value of `available` was the string `true` instead of a primitive boolean `true`.  We can always add a custom validator if we get sick of this string transform problem, but this will definitely work for now.

Thanks again!